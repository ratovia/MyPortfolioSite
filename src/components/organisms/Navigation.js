import React from "react"
import styled from "styled-components"
import { Menu } from "../molecules/Menu"

const Nav = styled.nav`
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: #f04949;
  text-orientation: sideways;
  writing-mode: vertical-rl;
  width: 100%;
  height: 100%;
  justify-content: center;
`

export const Navigation = () => {
  return (
    <Nav>
      <Menu></Menu>
    </Nav>
  )
}
