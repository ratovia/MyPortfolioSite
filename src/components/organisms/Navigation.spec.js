import React from "react"
import renderer from "react-test-renderer"
import { Navigation } from "./Navigation"
describe("Navigation", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Navigation />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
