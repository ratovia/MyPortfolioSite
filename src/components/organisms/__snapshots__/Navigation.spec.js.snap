// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders correctly 1`] = `
<nav
  className="sc-jRQAMF ckfUSk"
>
  <ul
    className="sc-eCImvq kJNpxb"
  >
    <li>
      <h3
        className="sc-hKwCoD NiySg"
      >
        <p
          className="sc-bdvvaa gkRUFT"
        >
          Top
        </p>
      </h3>
    </li>
    <li>
      <h3
        className="sc-hKwCoD NiySg"
      >
        <p
          className="sc-bdvvaa gkRUFT"
        >
          About
        </p>
      </h3>
    </li>
    <li>
      <h3
        className="sc-hKwCoD NiySg"
      >
        <p
          className="sc-bdvvaa gkRUFT"
        >
          Work
        </p>
      </h3>
    </li>
    <li>
      <h3
        className="sc-hKwCoD NiySg"
      >
        <p
          className="sc-bdvvaa gkRUFT"
        >
          Links
        </p>
      </h3>
    </li>
  </ul>
</nav>
`;
